mem_table.go

```diff
@@ -213,8 +216,10 @@ func (m *memTable) apply(batch *Batch, seqNum uint64) error {
 		}
 	}
 	if seqNum != startSeqNum+uint64(batch.Count()) {
-		panic(errors.Errorf("pebble: inconsistent batch count: %d vs %d",
-			errors.Safe(seqNum), errors.Safe(startSeqNum+uint64(batch.Count()))))
+		return errors2.CorruptionError{
```

Comment:
Safe to not panic since up in the stack the commit pipeline logs a fatal anyways.